frontend/src/components/common/Tab.jsx

```diff
@@ -15,10 +15,14 @@ class Tab extends Component {
     return (
       <span>
         {this.props.selected === UNSELECTED && (
-          <span style={{ color: "blue", cursor: "pointer" }}>{this.props.tabName}</span>
+          <button className={this.props.tabName} style={{ color: "blue", cursor: "pointer" }}>
```

Comment:
Can we move your styles object out of being inline and into a constant close to the top of the file? It means it's only defined once, and will make it easier if we ever decide to pull it into a css file.

Reply:
fixed in latest commit